.golangci.yml

```diff
@@ -30,6 +30,7 @@ linters:
     - gochecknoinits
     - dupl
     - goconst
+    - gocyclo
```

Comment:
please add an exception at the end of that yml instead of completly disabled gocyclo

Reply:
Sorry, didn't see there already were a bunch of those. Done!